contribs/gnodev/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/gnolang/gno/contribs/gnodev
 
-go 1.22
+go 1.22.0
```

Comment:
Weird.

Reply:
Without this additional `0`, I have this issue during the installation:` toolchain not available`
cc @gfanton

Reply:
It may have been updated since, but from this issue https://github.com/golang/go/issues/62278#issuecomment-1698829945, having 1.22 without the `patch` element is now considered a mistake. Following this, we should probably update all our `go.mod` files with `.0`.